cfn/auth0-collector.template

```diff
@@ -0,0 +1,809 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "Alert Logic template for creating a Auth0 Logs collector",
+    "Parameters": {
+        "AlertlogicAccessKeyId": {
+            "Description": "Alert Logic Access Key Id obtained from AIMS",
+            "Type": "String",
+            "Default": "5c850016c9eb6f3b"
```

Comment:
please remove default. It was used for testing.
I'll remove it for okta as well.